src/apis/RecommendedSubreddits.es6.js

```diff
@@ -0,0 +1,28 @@
+import { isEmpty } from 'lodash/lang';
+
+import apiRequest from '../apiBase/apiRequest';
+import Subreddit from '../models2/Subreddit';
+
+const parseBody = (res, apiResponse) => {
+  const { body } = res;
+
+  if (body.data && Array.isArray(body.data.children)) {
+    body.data.children.forEach(c => apiResponse.addResult(Subreddit.fromJSON(c.data)));
+    // sometimes, we get back empty object and 200 for invalid sorts like
+    // `mine` when logged out
+  } else if (!isEmpty(body)) {
+    apiResponse.addResult(Subreddit.fromJSON(body.data || body));
+  }
+
+  return apiResponse;
+};
+
+const get = (apiOptions, query) => {
+  return apiRequest(apiOptions, 'GET', 'api/similar_subreddits.json', { query })
+      .then(apiResponse => {
+        const { body: { json } } = apiResponse.response;
+        return parseBody(apiResponse.response, apiResponse);
```

Comment:
since `parseBody` is already taking in the `apiResponse`, it might be cleaner to have `parseBody` extract the response. Then your `return` statement could be a one liner, similar to [PostsEndpoint.js](https://github.com/reddit/node-api-client/blob/3X/src/apis/PostsEndpoint.js#L87)